src/components/Filters/FilterChecklist.vue

```diff
@@ -57,7 +57,7 @@
 
 <script>
 import LicenseExplanationTooltip from '~/components/Filters/LicenseExplanationTooltip'
-import VLicense from '~/components/License/VLicense'
+import VLicense from '~/components/License/VLicense.vue'
```

Comment:
You should not need to include the `.vue` extension in your imports.

Reply:
The "Go to Definition" doesn't work in VS Code without the extension, sadly, which is the main reason to add the import.

Reply:
I think I read Evan You's comment somewhere that it's recommended to use the extension.